libexec/rc/rc.conf

```diff
@@ -641,7 +641,7 @@ firstboot_sentinel="/firstboot"	# Scripts with "firstboot" keyword are run if
 
 # Emulation/compatibility services provided by /etc/rc.d/abi
 sysvipc_enable="NO"	# Load System V IPC primitives at startup (or NO).
-linux_enable="NO"	# Linux binary compatibility loaded at startup (or NO).
+linux_enable="YES"	# Linux binary compatibility loaded at startup (or NO).
```

Comment:
Another bikeshed option, but is this really needed for TrueOS or is it more important to stay in sync with FreeBSD on this? My opinion is to reset it back to "NO" and track FreeBSD.